packages/app-store/locations.ts

```diff
@@ -401,9 +401,9 @@ export function getSuccessPageLocationMessage(
   t: TFunction,
   bookingStatus?: BookingStatus
 ) {
-  const eventLocationType = getEventLocationType(location);
+  const eventLocationType = guessEventLocationType(location);
   let locationToDisplay = location;
-  if (eventLocationType && !eventLocationType.default && eventLocationType.linkType === "dynamic") {
+  if (eventLocationType && !eventLocationType.default) {
```

Comment:
This too. Also, this location stuff is complex and known to break quickly. I would prefer to have some tests.

I think it might help if we unit test this method?